src/platform/C4WindowGTK.cpp

```diff
@@ -378,6 +378,34 @@ static gboolean fullscreen_restore(gpointer data)
 	return FALSE;
 }
 
+static bool mouse_was_grabbed = false;
```

Comment:
Sure. I used a global variable because the fullscreen_restore code above does the same thing.